src/main/java/com/taxi/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,267 @@
+package com.taxi.dao.impl;
+
+import com.taxi.dao.CarDao;
+import com.taxi.exception.DataProcessingException;
+import com.taxi.model.Car;
+import com.taxi.model.Driver;
+import com.taxi.model.Manufacturer;
+import com.taxi.utils.DbConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class CarDaoImpl implements CarDao {
+    private DbConnectionUtil connectionUtil;
+
+    public CarDaoImpl(DbConnectionUtil connectionUtil) {
+        this.connectionUtil = connectionUtil;
+    }
+
+    @Override
+    public Car create(Car car) {
+        String createStatement = "INSERT INTO cars(model,manufacturer_id) VALUES (?,?);";
+        Connection connection = connectionUtil.getConnection();
+        try (PreparedStatement preparedStatement = connection.prepareStatement(createStatement,
+                Statement.RETURN_GENERATED_KEYS)) {
+            connection.setAutoCommit(false);
+            preparedStatement.setString(1, (car.getModel()));
+            preparedStatement.setLong(2, car.getManufacturer().getId());
+            preparedStatement.executeUpdate();
+            addDriversForCar(car,connection);
+            connection.commit();
```

Comment:
Давай сделаем по аналогии с Апдейтом
```suggestion
            connection.setAutoCommit(false);
            addCar(car,connection);;
            addDriversForCar(car,connection);
            connection.commit();
```